ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -141,8 +141,7 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
               <span className="utility-features">
                 <a href="#">
                   <i className="fa fa-pencil"></i>
-                </a>
-                <a data-toggle="modal" data-target="#confirm-delete"
+                </a> <a data-toggle="modal" data-target="#confirm-delete"
```

Comment:
what is the difference? The original spacing should be collapsed to 1 space, shouldn't it?